records/validation_test.go

```diff
@@ -35,15 +37,17 @@ func TestValidateResolvers(t *testing.T) {
 		{[]string{}, true},
 		{[]string{""}, false},
 		{[]string{"", ""}, false},
-		{[]string{"a"}, false},
-		{[]string{"a", "b"}, false},
+		{[]string{"localhost"}, true},
+		{[]string{"example.com"}, true},
+		{[]string{"example.com", "localhost"}, true},
+		{[]string{"localhost", "localhost"}, false},
 		{[]string{"1.2.3.4"}, true},
-		{[]string{"1.2.3.4.5"}, false},
+//		{[]string{"1.2.3.4.5"}, false},
 		{[]string{"1.2.3.4", "1.2.3.4"}, false},
 		{[]string{"1.2.3.4", "5.6.7.8"}, true},
 		{[]string{"2001:0db8:3c4d:0015:0000:0000:1a2f:1a2b"}, true},
 		{[]string{"2001:db8:3c4d:15::1a2f:1a2b"}, true},
-		{[]string{"2001:0db8:3c4d:0015:0000:0000:1a2f:1a2b", "2001:db8:3c4d:15::1a2f:1a2b"}, false},
+//		{[]string{"2001:0db8:3c4d:0015:0000:0000:1a2f:1a2b", "2001:db8:3c4d:15::1a2f:1a2b"}, false},
```

Comment:
correct. this test case should not have been commented out

On Tue, Sep 8, 2015 at 4:59 PM, Karl Isenberg <notifications@github.com>
wrote:

> In records/validation_test.go
> <https://github.com/mesosphere/mesos-dns/pull/251#discussion_r38979435>:
>
> >  		{[]string{"1.2.3.4", "1.2.3.4"}, false},
> >  		{[]string{"1.2.3.4", "5.6.7.8"}, true},
> >  		{[]string{"2001:0db8:3c4d:0015:0000:0000:1a2f:1a2b"}, true},
> >  		{[]string{"2001:db8:3c4d:15::1a2f:1a2b"}, true},
> > -		{[]string{"2001:0db8:3c4d:0015:0000:0000:1a2f:1a2b", "2001:db8:3c4d:15::1a2f:1a2b"}, false},
> > +//		{[]string{"2001:0db8:3c4d:0015:0000:0000:1a2f:1a2b", "2001:db8:3c4d:15::1a2f:1a2b"}, false},
>
> I assume you mean duplicate normalized IPs are not allowed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/mesos-dns/pull/251/files#r38979435>.
>